src/earn/EarnEnterAmount.tsx

```diff
@@ -0,0 +1,293 @@
+    ValoraAnalytics.track(EarnEvents.earn_enter_amount_info_press)
+    infoBottomSheetRef.current?.snapToIndex(0)
+  }
+
+  return (
+    <EnterAmount
+      tokens={[token]}
+      defaultToken={token}
+      prepareTransactionsResult={prepareTransactionsResult}
+      onClearPreparedTransactions={clearPreparedTransactions}
+      onRefreshPreparedTransactions={handleRefreshPreparedTransactions}
+      prepareTransactionError={prepareTransactionError}
+      tokenSelectionDisabled={true}
+      onPressProceed={onPressContinue}
+      onPressInfo={onPressInfo}
+      onSetTokenAmount={(amount: BigNumber) => setTokenAmount(amount)}
+      ProceedComponent={EarnProceed}
+      proceedComponentStatic={true}
+      disableBalanceCheck={true}
+    >
+      <InfoBottomSheet infoBottomSheetRef={infoBottomSheetRef} />
+      <EarnAddCryptoBottomSheet
+        forwardedRef={addCryptoBottomSheetRef}
+        token={token}
+        tokenAmount={tokenAmount.minus(token.balance)}
+      />
+      {prepareTransactionsResult && prepareTransactionsResult.type === 'possible' && (
+        <EarnDepositBottomSheet
+          forwardedRef={reviewBottomSheetRef}
+          preparedTransaction={prepareTransactionsResult}
+          amount={tokenAmount.toString()}
+          tokenId={token.tokenId}
+        />
+      )}
+    </EnterAmount>
+  )
+}
+
+function EarnProceed({
+  tokenAmount,
+  localAmount,
+  token,
+  amountEnteredIn,
+  disabled,
+  onPressProceed,
+  onPressInfo,
+}: ProceedComponentProps) {
+  const { t } = useTranslation()
+  const localCurrencySymbol = useSelector(getLocalCurrencySymbol)
+
+  const asyncPoolInfo = useAsync(
+    async () => {
+      if (!token || !token.address) {
+        throw new Error(`Token with id ${token} not found`)
+      }
+
+      if (!isAddress(token.address)) {
+        throw new Error(`Token with id ${token} does not contain a valid address`)
+      }
+
+      return fetchAavePoolInfo({
+        assetAddress: token.address,
+        contractAddress: networkConfig.arbAavePoolV3ContractAddress,
+        network: networkIdToNetwork[token.networkId],
+      })
+    },
+    [],
+    {
+      onError: (error) => {
+        Logger.warn(TAG, error.message)
+      },
+    }
+  )
+
+  return (
+    <View style={styles.infoContainer}>
+      <View style={styles.line}>
+        <Text style={styles.label}>{t('earnFlow.enterAmount.earnUpToLabel')}</Text>
+        <Text style={styles.label}>{t('earnFlow.enterAmount.rateLabel')}</Text>
+      </View>
+      <View style={styles.line}>
+        <Text style={styles.valuesText} testID="EarnEnterAmount/EarnUpTo">
+          {t('earnFlow.enterAmount.earnUpTo', {
+            fiatSymbol: localCurrencySymbol,
+            amount:
+              asyncPoolInfo?.result && !!asyncPoolInfo.result.apy && tokenAmount?.gt(0)
+                ? tokenAmount.multipliedBy(new BigNumber(asyncPoolInfo.result.apy)).toFormat(2)
+                : '--',
+          })}
+        </Text>
```

Comment:
I guess the design linked in the ticket shows TVL here, is this from the follow up? Just confirming, should be fine to leave this as is

Reply:
yeah talking with Jacob we decided to just show the earn up to amount here, as it seemed easier than getting TVL which would then just be removed